codegen-renderers/src/test/kotlin/io/vrap/codegen/languages/TestCodeGenerator.kt

```diff
@@ -42,6 +52,66 @@ class TestCodeGenerator {
         generatorComponent.generateFiles()
     }
 
+    @Test
+    fun generateJavaModelsWithInterfacesModule() {
+        val generatorConfig = CodeGeneratorConfig(basePackageName = "com.commercetools.importer", outputFolder = Paths.get("build/gensrc/java"))
+        val generatorModule = GeneratorModule(apiProvider, generatorConfig, JavaBaseTypes)
+        val generatorComponent = GeneratorComponent(generatorModule, JavaCompleteModule())
+        generatorComponent.generateFiles()
+        
+        File("build/compiled").deleteRecursively()
+        File("build/compiled").mkdirs()
+        
+        val filesList =  File("build/gensrc")
+            .walkTopDown()
+            .filter( File::isFile )
+            .filter { it.name.endsWith(".java") }
+            .toList()
+
+        val compiler = ToolProvider.getSystemJavaCompiler()
+        val fileManager = compiler.getStandardFileManager(null, null, null)
+
+        fileManager.use {
+            it.setLocation(StandardLocation.CLASS_OUTPUT, Arrays.asList(File("build/compiled")))
+            val result = compiler.getTask(null,
+                    fileManager, null, null, null,
+                    fileManager.getJavaFileObjectsFromFiles(filesList)).call()
+
+            Assert.assertTrue(result)
+        }
+    }
+
+    @Test
+    fun generateJavaCompleteModule() {
+        val generatorConfig = CodeGeneratorConfig(basePackageName = "com.commercetools.importer", outputFolder = Paths.get("../generated-code/src/test/java"))
+        val generatorModule = GeneratorModule(apiProvider, generatorConfig, JavaBaseTypes)
+        val generatorComponent = GeneratorComponent(generatorModule, JavaCompleteModule())
+        generatorComponent.generateFiles()
+    }
+
+    /**
+     * This test method uses code generator to generate interface and a class for simple-type.raml which is a part of the test-api.raml located in the resources
+     * folder. After the classes are generated, it checks if they are the same as the ones specified in SimpleType.txt and SimpleTypeImpl.txt.
+     */
+    @Ignore
```

Comment:
Because the generated model classes are being changed daily and the test loses its purpose as every time I just overwrite the whole generated txt file without even looking at what changed. Also, the new model has a more complex structure that should be tested once we are sure that it won't change much in the future. So right now the test fails so I change the test, doesn't feel right.